backend/src/extension.ts

```diff
@@ -159,7 +165,11 @@ export class YeomanUIPanel {
 			indexHtml = indexHtml.replace(/<script src=/g, `<script src=${scriptUri.toString()}`);
 			indexHtml = indexHtml.replace(/<img src=/g, `<img src=${scriptUri.toString()}`);
 		}
-		this.panel.title = 'Yeoman UI';
+		const uiMessages = _.assign(backendMessages, _.get(YeomanUIPanel, "messages", {}));
```

Comment:
move this line to extension constructor

Reply:
I will probably do it after i get some information from Avital.
Currently we have a bug when you try to run one of two existings commands.
After Avital answers on my questions, i will decide whether to move the code to constructor, as you wrote, or leave it as is